### ISSUE TITLE
New Toggle Icon is cluttered

### ISSUE THREAD

**Issue by markmonroy** [johnholbrook/streamdeck-pihole#19]
Hi! Thanks for the plugin. With the recent update the toggle icon added an on/off switch to the lower right. Unfortunately this makes things look quite cluttered with the addition of Ads Blocked Today, and is also redundant because I could easily tell by the absence of color that Pi-Hole was toggled off. I think I'm smart enough to rescue the icons from an earlier version and load them in manually, but I just wanted to give my 2 cents on the change and see if anyone else agrees.
![Screenshot 2023-07-25 at 8 53 24 AM](https://github.com/johnholbrook/streamdeck-pihole/assets/32033503/c8879236-1b0c-4bbc-97a6-361e30f2c462)

**Comment by AcidWeb** [comment]
Yeah. It looks worse.

**Comment by rtheil** [comment]
Agreed. I think it was fine without the toggle icon since the entire icon grayed out makes it obvious it's disabled.

**Comment by t1m0thyj** [comment]
+1 - I use the toggle action and would like to show % Ads Blocked Today but having white numbers overlaid on a white toggle symbol looks jarring.
